pwa/src/templates/components/ComponentsTemplate.module.css

```diff
@@ -18,8 +20,11 @@
   margin-inline-end: var(--nlportal-space-block-xl);
 }
 
-.componentsGrid {
-  display: grid;
-  grid-gap: var(--nlportal-space-block-md);
-  grid-template-columns: 1fr 1fr;
+.buttongroup {
```

Comment:
`.buttonGroup`*

Also, maybe we could make it a bit more declarative. `layoutSwitchButtonsContainer`? I'm not sure.